From b252d80468317b6e286017dab86aef2dbc08d51f Mon Sep 17 00:00:00 2001
From: Andrew Wang <wang@apache.org>
Date: Fri, 19 Sep 2014 17:42:00 -0700
Subject: [PATCH 425/551] HADOOP-11112. TestKMSWithZK does not use KEY_PROVIDER_URI. (tucu via wang)

(cherry picked from commit b6ceef90e52c05fd95bfb40bff61fa7e1802787f)
(cherry picked from commit eec927f3fcce172aee82a6c51f358fec86e6ba47)
---
 .../crypto/key/kms/server/TestKMSWithZK.java       |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMSWithZK.java b/hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMSWithZK.java
index 59b0002..3a02a0a 100644
--- a/hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMSWithZK.java
+++ b/hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMSWithZK.java
@@ -76,7 +76,7 @@
 
   protected Configuration createBaseKMSConf(File keyStoreDir) throws Exception {
     Configuration conf = new Configuration(false);
-    conf.set("hadoop.security.key.provider.path",
+    conf.set(KMSConfiguration.KEY_PROVIDER_URI,
         "jceks://file@" + new Path(keyStoreDir.getAbsolutePath(),
             "kms.keystore").toUri());
     conf.set("hadoop.kms.authentication.type", "simple");
-- 
1.7.0.4

